.travis.yml

```diff
@@ -10,6 +10,7 @@ addons:
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml # because pip installation is slow
+      - unixodbc-dev
```

Comment:
Why is it required?

Reply:
Thierry added it to fix tests on py3. No traceback tho. I removed it, let's see how travis behaves...